_data/companies.yml

```
@@ -117,6 +117,13 @@
   events: Restricted
   last_update: 2020-03-12
 
+- name: Bitdefender
+  wfh: Required
+  travel: Restricted
+  visitors: Restricted
+  events: ?
```

```suggestion
  events: "?"
```

Question marks need to be in quotes in YAML.